src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetApprenticeship/GetApprenticeshipQueryHandler.cs

```diff
@@ -62,13 +62,16 @@ public async Task<GetApprenticeshipQueryResult> Handle(GetApprenticeshipQuery re
                             : default(long?),
                         OriginalStartDate = apprenticeship.OriginalStartDate,
                         ApprenticeshipEmployerTypeOnApproval = apprenticeship.Cohort.ApprenticeshipEmployerTypeOnApproval,
-                        MadeRedundant = apprenticeship.MadeRedundant
+                        MadeRedundant = apprenticeship.MadeRedundant,                        
+                        ConfirmationStatus = apprenticeship.DisplayConfirmationStatus(
+                            apprenticeship.Email,
+                            apprenticeship.ApprenticeshipConfirmationStatus != null ? apprenticeship.ApprenticeshipConfirmationStatus.ApprenticeshipConfirmedOn : null,
+                            apprenticeship.ApprenticeshipConfirmationStatus != null ? apprenticeship.ApprenticeshipConfirmationStatus.CommitmentsApprovedOn : DateTime.MinValue,
+                            apprenticeship.ApprenticeshipConfirmationStatus != null ? apprenticeship.ApprenticeshipConfirmationStatus.ConfirmationOverdueOn : null)
```

Comment:
It's hard to read this (not your fault), but there may be a slightly better way to do this 

ConfirmationStatus = apprenticeship.DisplayConfirmationStatus(
                            apprenticeship.Email,
                            apprenticeship.ApprenticeshipConfirmationStatus?.ApprenticeshipConfirmedOn,
                            apprenticeship.ApprenticeshipConfirmationStatus?.ConfirmationOverdueOn)
                            
it might be possible to create as an Expression function, but I need to look into that a little